tenbilac/com.py

```diff
@@ -149,14 +164,14 @@ def train(self, inputs, targets, inputnames=None, targetnames=None):
 					multactfctname=self.config.get("net", "multactfctname"),
 					inames=inputnames,
 					onames=targetnames,
-					name='{}-{}-{}'.format(nettype, self.name, i)
+					name='{}_{}'.format(self.name, i)
 					)
 			else:
 				raise RuntimeError("Don't know network type '{}'".format(nettype))
 			
 			
 			# A directory where the training can store its stuff
-			trainobjdir = os.path.join(self.workdir, "{}_{:03d}".format(self.name, i))
+			trainobjdir = os.path.join(self.workdir, "{}_{}".format("member", i))
```

Comment:
`"{}_{}".format("member", i)` it's a bit of an overkill, no? better would be: `"member_{}".format(i)`